HackerNewsWrapperApi/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using HackerNewsWrapperApi.Services;
+using Polly;
+using Serilog;
+
+namespace HackerNewsWrapperApi.Extensions;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddCustomHttpClient(this IServiceCollection services)
+    {
+        var timeoutPolicy = Policy.TimeoutAsync<HttpResponseMessage>(10);
+        services.AddHttpClient<HackerHttpService>()
```

Comment:
Think about how to rename and separate HttpClient logic and your Service. It can not be called HackerHttpService.